test/test_serialization_utils.py

```diff
@@ -58,3 +59,10 @@ def test_serialize_timestamp(self):
         datetime_object = datetime.datetime.strptime(datetime_str, "%Y-%m-%d")
         result = _serialize_json_safe(datetime_object)
         assert datetime_str in result
+
+    def test_serialize_via_json_timestamp(self):
+        timestamp_obj = pd.Timestamp(2020, 1, 1)
+        assert isinstance(timestamp_obj, pd._libs.tslibs.timestamps.Timestamp)
+        result = _serialize_json_safe(timestamp_obj)
```

Comment:
also maybe validate if it is in a list or numpy array if the serialization works?

Reply:
Done